lib/streamlit/hashing.py

```diff
@@ -406,6 +407,12 @@ def _to_bytes(self, obj, context):
             self.update(h, obj.tell())
             return h.digest()
 
+        elif isinstance(obj, Pattern):
+            h = hashlib.new("md5")
+            self.update(h, obj.pattern)
+            self.update(h, obj.flags)
+            return h.digest()
```

Comment:
You can just return `b"%s:%s" % (obj.pattern, obj.flags)` here since it gets md5'ed inside `update()`

Reply:
Perhaps it should be return `self.to_bytes((obj.pattern, obj.flags,))`?

Your suggestion also works but we only use `%s:%s` when we prefix with the `type` at the moment.